src/Interfaces/IGlacisAirlift.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+/// @custom:version 1.0.0
+pragma solidity ^0.8.17;
+
+struct QuoteSendInfo {
+    Fee gmpFee;
+    uint256 amountSent;
+    uint256 valueSent;
+    AirliftFeeInfo airliftFeeInfo;
+}
+
+struct AirliftFeeInfo {
+    Fee airliftFee;
+    uint256 correctedAmount;
+    uint256 correctedValue;
+}
+
+struct Fee {
+    uint256 nativeFee;
+    uint256 tokenFee;
+}
+
+error GlacisAirlift__NotEnoughValueFee();
```

Comment:
We dont need all these errors, etc.
Only put code in our codebase that is required, keep interfaces as small as (reasonably) possible cause this bloats up our codebase

Reply:
ok!